automated-testing/readme.md

```diff
@@ -0,0 +1,16 @@
+# Testing
+
+## Sections within Testing
+
+* [Unit testing](unit-testing/readme.md)
+* [Unit testing v2](unit-testing-v2/readme.md)
+* [Integration testing](integration-testing/readme.md)
```

Comment:
This is a LOT of different types of testing, getting content for all of these would definitely be our end goal, however I am wondering whether the focus should be on getting a few of these (most used) done first vs putting placeholders/templates for each of these.

Reply:
+1 - I don't think we should check in the template as content for each readme. Building this incrementally makes sense as well vs putting placeholders.

Reply:
Thank you for the feedback. The idea of the whole PR was to create placeholder pages based on the template we picked. I am ok to modify the approach. Let's sync today and decide on our strategy. 

Thank you!